### ISSUE TITLE
Add a "tree" view

### ISSUE THREAD

**Issue by coxley** [etsy/hound#201]
Hound is super simple to deploy, but hundreds of repos to search through it gets quite hard to parse. I suggest having a view to "minimize" the repo names where matches were found and let the user expand them?

**Comment by dcampbell24** [comment]
If I understand you, that sounds like a super helpful feature to have. The results would start off as just the book icons with the repo name next to it (maybe add some stats like how many matches were in the repo) and then clicking on the book icon would cause results to be expanded. The icon could be clicked on further times to toggle back and forth and would probably display a different looking book (open or closed).

I think for when files have a lot of matches it might be nice if the lines within the file where the matches were started hidden and then could be toggled. There might also be some expand and collapse all buttons.

**Comment by coxley** [comment]
That about sums up my thoughts. :)

**Comment by danielcb** [comment]
#149

**Comment by mikepurvis** [comment]
I'm interested in this feature too; we also have 100+ repositories at our org.

Has anyone done an implementation?

**Comment by grahamhayes** [comment]
I have found `http <server>/api/v1/search q=<query> repos='*' -f | jq '.Results | keys'` using [httpie](https://httpie.org/) and [jq](https://stedolan.github.io/jq/) (should work fine with curl as well) gets the name of the repos.

e.g. 
```
➜  designate git:(master) http http://codesearch.openstack.org/api/v1/search q=ujson repos='*' files="requirements\.txt" -f | jq '.Results | keys' 
[
  "kiloeyes",
  "monasca-common",
  "requirements",
  "rpm-packaging"
]
```
